src/backend/utils/misc/guc.c

```diff
@@ -8200,7 +8200,7 @@ static struct config_string ConfigureNamesString[] =
 			NULL
 		},
 		&standby_addr_host,
-		"localhost", NULL, NULL
+		"none", NULL, NULL
```

Comment:
I suspect this does not affect your code.

See the code below,

def get_all_values(self):
        if 'hawq_standby_address_host' in self.hawq_dict:
            if self.hawq_dict['hawq_standby_address_host'].lower() in ['none', '', 'localhost']:
                del self.hawq_dict['hawq_standby_address_host']

However hawq_ctl should remove the "none" case, I think.

@radarwave Please confirm the logic.